### ISSUE TITLE
Prod package for Amazon Linux 2 distro not found

### ISSUE THREAD

**Issue by khoaxtru20** [microsoft/linux-package-repositories#84]
**Describe the issue**

[https://packages.microsoft.com/amazonlinux/2/prod/](https://packages.microsoft.com/amazonlinux/2/prod/) returns "404 Not Found".

**When did the issue occur?**

Happened while trying to install Azure Arc Agent for Amazon Linux 2 machine. Followed these instructions: [https://learn.microsoft.com/en-us/linux/packages#red-hat-based-linux-distributions](https://learn.microsoft.com/en-us/linux/packages#red-hat-based-linux-distributions) with yum.

**Actual Result**

After attempting to run `sudo run update`, got the following error: `failure: repodata/repomd.xml from packages-microsoft-com-prod: [Errno 256] No more mirrors to try.
https://packages.microsoft.com/amazonlinux/2/prod/repodata/repomd.xml: [Errno 14] HTTPS Error 404 - Not Found`

**Comment by khoaxtru20** [comment]
Workaround:

Got the distro that my OS is based off of with `cat /proc/version`. I found that mine is based off `RHEL 7.3`

I tried to run through the instructions again with `rhel/7.3` but that [config file is missing](https://packages.microsoft.com/config/rhel/). Instead, I edited the config file at `/etc/yum.repos.d/microsoft-prod.repo` which I had already installed after running through the instructions the first time. I changed the baseurl to point at `https://packages.microsoft.com/rhel/7.3/`.

Ran `sudo yum update` then `sudo yum install <package>`

Ran the package and followed the instructions to connect the device. Seems like a success so far.

And in case anyone else is in my exact situation, I later did a google search and found this article to [install the Azure Arc agent](https://learn.microsoft.com/en-us/azure/network-watcher/connection-monitor-connected-machine-agent?tabs=LinuxScript).... Although I haven't tried it out, it probably would've been way less of a headache....
